### ISSUE TITLE
backend: provide existing schemas to frontend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#54]
## User story
1. As a {HR-employee}
2. I want / need {the DIdentity backend to be able to get all existing schemas}
3. So that {they can later be displayed to me in a schema-overview-page}

## Acceptance criteria
* all existing schemas can be requested by the frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
